model-building.Rmd

```diff
@@ -399,72 +399,68 @@ daily %>%
+    para sábados, es decir, debería tener `Thurs`, `Fri`, pero `Sat-summer`, 
+    `Sat-spring`, `Sat-fall`. ¿Cómo este modelo se compara con el modelo que tiene 
+    la combinación de `wday` y `term`?
     
-1.  Create a new `wday` variable that combines the day of week, term 
-    (for Saturdays), and public holidays. What do the residuals of 
-    that model look like?
+1.  Crea una nueva variable `wday` que combina el día de la semana, términos 
+    (para sábados), y feriados públicos. ¿a qué se parecen los residuos 
+    del modelo?
 
-1.  What happens if you fit a day of week effect that varies by month 
-    (i.e. `n ~ wday * month`)? Why is this not very helpful? 
+1.  ¿Qué sucede si ajustamos un efecto de día de la semana que varía mes por mes 
+    (es decir, `n ~ wday * month`)? ¿Por qué esto no es muy útil? 
 
-1.  What would you expect the model `n ~ wday + ns(date, 5)` to look like?
-    Knowing what you know about the data, why would you expect it to be
-    not particularly effective?
-
-1.  We hypothesised that people leaving on Sundays are more likely to be 
-    business travellers who need to be somewhere on Monday. Explore that 
-    hypothesis by seeing how it breaks down based on distance and time: if 
-    it's true, you'd expect to see more Sunday evening flights to places that 
-    are far away.
+1.  ¿Que esperarías del modelo `n ~ wday + ns(date, 5)`?
+    Sabiendo lo que sabes sobre los datos, ¿porqué esperarias que no sea 
+    particularmente efectivo?
+    
+1.  Presumimos que las personas que salen los domingos son probablemente 
+    viajeros de negocios quienes necesitan estar en algun lugar el lunes. Explora esa 
+    hipótesis al ver cómo se rompe en función de la distancia y tiempo: si 
+    es verdad, esperarías ver más vuelos en la tarde del domingo a lugares que estan muy lejos.
 
-1.  It's a little frustrating that Sunday and Saturday are on separate ends
-    of the plot. Write a small function to set the levels of the 
-    factor so that the week starts on Monday.
+1.  Es un poco frustante que el domingo y sábado tengan fines separados
+    del gráfico. Escribe una pequeña función para establecer los niveles del 
+    factor para que la semana comience el lunes.
 
-## Learning more about models
+## Aprende más sobre los modelos
 
-We have only scratched the absolute surface of modelling, but you have hopefully gained some simple, but general-purpose tools that you can use to improve your own data analyses. It's OK to start simple! As you've seen, even very simple models can make a dramatic difference in your ability to tease out interactions between variables.
+Solo hemos dado una pincelada acerca de los modelos, pero es de esperar que hayas obtenido algunas herramientas simples, pero de uso general que puedes usar para mejorar tus propios análisis. !Está bien para empezar! Como has visto, incluso modelos muy simples pueden marcar una gran capacidad para desentrañar interacciones entre variables.
```

Comment:
traducir *It's OK to start simple!* como *¡Está bien empezar de forma sencilla!* o ¡Está bien empezar simple!*

Reply:
traducir *As you've seen, even very simple models can make a dramatic difference in your ability to tease out interactions between variables* como *Como has visto, incluso modelos muy simples pueden **producir una gran diferencia en tu capacidad para aclarar/distinguir las interacciones** entre variables.*